addons/web_editor/static/src/xml/add_snippet_dialog.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+
+<t t-name="web_editor.AddSnippetDialog">
+    <Dialog
+        title="title"
+        contentClass="'o_add_snippet_dialog h-100'"
+        footer="false"
+        size="'xl'">
+        <div class="overflow-hidden w-100">
+            <div class="d-flex w-100 h-100 vertical flex-row">
+                <ul class="nav nav-pills flex-column flex-nowrap overflow-y-auto border-end" t-ref="snippetGroupTabs">
+                <div class="input-group p-2 bg-100 border-bottom">
+                    <input type="search" class="form-control bg-white" placeholder="Search for a block"
+                        aria-label="Search for a block" t-model="state.search" t-ref="search-input"/>
+                    <span class="input-group-text border-0">
+                        <i class="fa fa-search"></i>
```

Comment:
There's an `aria-label` on the input, so this decorative icon should be made `aria-hidden="true"`.